### ISSUE TITLE
[a11y] Accessibility - EnablementDialog Enable the required activity? cannot be read by JAWS

### ISSUE THREAD

**Issue by GeraldMit** [eclipse-platform/eclipse.platform.ui#272]
https://github.com/eclipse-platform/eclipse.platform.ui/blob/04cb8c2b5db6f80f5bb75fdbe9270478c9c9c406/bundles/org.eclipse.ui.workbench/Eclipse%20UI/org/eclipse/ui/internal/activities/ws/EnablementDialog.java#L148

https://github.com/eclipse-platform/eclipse.platform.ui/blob/04cb8c2b5db6f80f5bb75fdbe9270478c9c9c406/bundles/org.eclipse.ui.workbench/Eclipse%20UI/org/eclipse/ui/internal/activities/ws/EnablementDialog.java#L156

JAWS likely doesn't read it out because labels are supposed to be labels for text fields according to JAWS 
looking at the code, line 148 is where a second Label is created right after the first is where the "Enable the required activity?" string is introduced to the UI.

**Comment by niraj-modi** [comment]
This Label doesn't grab tab focus, hence is not read by Accessible tools like JAWS.
Hence nesting it inside a composite and setting an explicit TabList works.

Will share a fix patch for this shortly.

**Comment by niraj-modi** [comment]
Verified the fix in Eclipse Build id: I20220824-1800 on Win10 using JAWS-2021

**Comment by niraj-modi** [comment]
Fix is back-ported to below branches:
- R4_23_maintenance
- R4_8_maintenance
